src/snapred/backend/dao/calibration/CalibrationRecord.py

```diff
@@ -17,5 +17,5 @@ class CalibrationRecord(BaseModel):
     calibrationFittingIngredients: Calibration
     pixelGroups: Optional[List[PixelGroup]]  # TODO: really shouldnt be optional, will be when sns data fixed
     focusGroupCalibrationMetrics: FocusGroupMetric
-    workspaceNames: List[WorkspaceName]
+    workspaces: Dict[str, WorkspaceName]
```

Comment:
I'm tempted to say this should instead be its own Pydantic Model instead of a dict, the only downside to that I'm seeing is if at somepoint we abstracted the record as a whole to support N sets of metadata and M workspaces being saved.

Reply:
**I modified this treatment only slightly, with the argument that it's a temporary work around in any case and we shouldn't spend too much time on it right now.**  Malcolm just asked us to provide diffraction-calibration output workspaces, in TOF and DSP units simultaneously.  With this in mind, at each WNG.WorkspaceType as key, there is now a list of workspace names.